ui/apps/platform/src/Containers/Clusters/cluster.helpers.tsx

```diff
@@ -23,9 +23,9 @@ export const runtimeOptions = [
         value: 'NO_COLLECTION',
     },
     {
-        label: 'Kernel Module',
-        tableDisplay: 'Kernel Module',
-        value: 'KERNEL_MODULE',
+        label: 'CORE BPF',
+        tableDisplay: 'CORE BPF',
+        value: 'CORE_BPF',
```

Comment:
Forgot to add core bpf here in an earlier PR.

Reply:
Aha, looking with fresh eyes, `tableDisplay` is obsolete in `runtimeOptions` and `clusterTypeOptions`

UI team can follow up to delete after 4.1 release when no risk of merge conflicts.